hostname sb048.cluster
ipconfig 1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000 link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00 inet 127.0.0.1/8 scope host lo valid_lft forever preferred_lft forever inet6 ::1/128 scope host valid_lft forever preferred_lft forever 2: enp4s0f0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc mq state UP group default qlen 1000 link/ether a0:8c:f8:34:c9:d8 brd ff:ff:ff:ff:ff:ff inet 10.3.81.128/22 brd 10.3.83.255 scope global enp4s0f0 valid_lft forever preferred_lft forever inet6 fe80::a28c:f8ff:fe34:c9d8/64 scope link valid_lft forever preferred_lft forever 3: enp4s0f1: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:34:c9:d9 brd ff:ff:ff:ff:ff:ff 4: enp4s0f2: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:34:c9:da brd ff:ff:ff:ff:ff:ff 5: enp4s0f3: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000 link/ether a0:8c:f8:34:c9:db brd ff:ff:ff:ff:ff:ff 6: ib0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 4092 qdisc pfifo_fast state UP group default qlen 256 link/infiniband 80:00:00:29:fe:80:00:00:00:00:00:00:38:bc:01:03:00:4c:f4:00 brd 00:ff:ff:ff:ff:12:40:1b:ff:ff:00:00:00:00:00:00:ff:ff:ff:ff inet 10.3.89.128/22 brd 10.3.91.255 scope global ib0 valid_lft forever preferred_lft forever inet 10.3.97.128/22 brd 10.3.99.255 scope global ib0:1 valid_lft forever preferred_lft forever inet6 fe80::3abc:103:4c:f400/64 scope link valid_lft forever preferred_lft forever 7: ib1: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 4092 qdisc pfifo_fast state DOWN group default qlen 256 link/infiniband 80:00:00:29:fe:80:00:00:00:00:00:00:38:bc:01:03:00:4c:f4:01 brd 00:ff:ff:ff:ff:12:40:1b:ff:ff:00:00:00:00:00:00:ff:ff:ff:ff
2024-01-29 02:30:41,736 - distributed.nanny - INFO -         Start Nanny at: 'tcp://10.3.89.128:40363'
2024-01-29 02:30:45,681 - distributed.diskutils - INFO - Found stale lock file and directory '/scratch/dask-worker-space/worker-jlltk2wz', purging
2024-01-29 02:30:53,886 - distributed.worker - INFO -       Start worker at:    tcp://10.3.89.128:38395
2024-01-29 02:30:53,889 - distributed.worker - INFO -          Listening to:    tcp://10.3.89.128:38395
2024-01-29 02:30:53,889 - distributed.worker - INFO -           Worker name:           SLURMCluster-357
2024-01-29 02:30:53,889 - distributed.worker - INFO -          dashboard at:          10.3.89.128:38659
2024-01-29 02:30:53,889 - distributed.worker - INFO - Waiting to connect to:     tcp://10.3.88.11:44532
2024-01-29 02:30:53,890 - distributed.worker - INFO - -------------------------------------------------
2024-01-29 02:30:53,890 - distributed.worker - INFO -               Threads:                          2
2024-01-29 02:30:53,890 - distributed.worker - INFO -                Memory:                  20.49 GiB
2024-01-29 02:30:53,890 - distributed.worker - INFO -       Local Directory: /scratch/dask-worker-space/worker-3w4vpilq
2024-01-29 02:30:53,890 - distributed.worker - INFO - -------------------------------------------------
2024-01-29 02:30:53,948 - distributed.worker - INFO -         Registered to:     tcp://10.3.88.11:44532
2024-01-29 02:30:53,948 - distributed.worker - INFO - -------------------------------------------------
2024-01-29 02:30:53,949 - distributed.core - INFO - Starting established connection to tcp://10.3.88.11:44532
2024-01-29 02:52:40,163 - distributed.worker - ERROR - Worker stream died during communication: tcp://10.3.89.129:46774
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.129:46774 after 30 s
2024-01-29 02:53:11,955 - distributed.worker - ERROR - Worker stream died during communication: tcp://10.3.89.129:46774
Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/tcp.py", line 225, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 490, in wait_for
    return fut.result()
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 328, in connect
    handshake = await asyncio.wait_for(comm.read(), time_left())
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/asyncio/tasks.py", line 492, in wait_for
    raise exceptions.TimeoutError() from exc
asyncio.exceptions.TimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2081, in gather_dep
    response = await get_data_from_worker(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/worker.py", line 2900, in get_data_from_worker
    comm = await rpc.connect(worker)
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1468, in connect
    return await connect_attempt
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/core.py", line 1389, in _connect
    comm = await connect(
  File "/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/distributed/comm/core.py", line 333, in connect
    raise OSError(
OSError: Timed out during handshake while connecting to tcp://10.3.89.129:46774 after 30 s
2024-01-29 02:53:37,670 - distributed.core - INFO - Event loop was unresponsive in Worker for 23.26s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-01-29 02:53:37,670 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 23.26s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-01-29 02:53:50,751 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 7)

2024-01-29 02:53:53,664 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 7)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/__init__.py:55: UserWarning: Dependency not satisfied, torchani.ase will not be available
  warnings.warn("Dependency not satisfied, torchani.ase will not be available")
2024-01-29 02:54:38,656 [WARNING] [__init__.py:5] root: Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 7 conformers. 
Removed 4 conformers. 
Generated 46 conformers. 
Removed 39 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/3 [00:00<?, ?it/s][W BinaryOps.cpp:601] Warning: floor_divide is deprecated, and will be removed in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values.
To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor'). (function operator())
Optimising conformer:  33%|███████▋               | 1/3 [00:05<00:10,  5.33s/it]Optimising conformer:  67%|███████████████▎       | 2/3 [00:07<00:03,  3.32s/it]Optimising conformer: 100%|███████████████████████| 3/3 [00:10<00:00,  3.41s/it]Optimising conformer: 100%|███████████████████████| 3/3 [00:10<00:00,  3.59s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-plip/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
TIME Completed the molecule generation in 82.5s.
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/7 [00:00<?, ?it/s]Optimising conformer:  14%|███▎                   | 1/7 [00:03<00:23,  3.85s/it]Optimising conformer:  29%|██████▌                | 2/7 [00:07<00:19,  3.93s/it]Optimising conformer:  43%|█████████▊             | 3/7 [00:11<00:14,  3.74s/it]Optimising conformer:  57%|█████████████▏         | 4/7 [00:15<00:12,  4.01s/it]Optimising conformer:  71%|████████████████▍      | 5/7 [00:19<00:07,  3.97s/it]Optimising conformer:  86%|███████████████████▋   | 6/7 [00:22<00:03,  3.59s/it]Optimising conformer: 100%|███████████████████████| 7/7 [00:27<00:00,  4.03s/it]Optimising conformer: 100%|███████████████████████| 7/7 [00:27<00:00,  3.93s/it]
2024-01-29 02:55:33,911 - distributed.utils_perf - INFO - full garbage collection released 12.49 MiB from 420914 reference cycles (threshold: 9.54 MiB)
2024-01-29 02:55:33,911 [INFO] [utils_perf.py:198] distributed.utils_perf: full garbage collection released 12.49 MiB from 420914 reference cycles (threshold: 9.54 MiB)
2024-01-29 03:12:32,623 - distributed.core - INFO - Event loop was unresponsive in Worker for 4.17s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-01-29 03:12:32,623 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 4.17s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-01-29 03:12:55,416 - distributed.core - INFO - Event loop was unresponsive in Worker for 3.94s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2024-01-29 03:12:55,416 [INFO] [core.py:595] distributed.core: Event loop was unresponsive in Worker for 3.94s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-01-29 03:33:41,430 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 11)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-01-29 03:33:43,477 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 11)

TIME Completed the molecule generation in 118.1s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 8 conformers. 
Removed 6 conformers. 
Generated 50 conformers. 
Removed 41 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/9 [00:00<?, ?it/s]Optimising conformer:  11%|██▌                    | 1/9 [00:04<00:34,  4.26s/it]Optimising conformer:  22%|█████                  | 2/9 [00:09<00:32,  4.67s/it]Optimising conformer:  33%|███████▋               | 3/9 [00:15<00:31,  5.21s/it]Optimising conformer:  44%|██████████▏            | 4/9 [00:28<00:42,  8.44s/it]Optimising conformer:  56%|████████████▊          | 5/9 [00:31<00:26,  6.60s/it]using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'

Optimising conformer:   0%|                               | 0/2 [00:00<?, ?it/s][A
Optimising conformer:  50%|███████████▌           | 1/2 [00:03<00:03,  3.40s/it][A
Optimising conformer: 100%|███████████████████████| 2/2 [00:05<00:00,  2.57s/it][AOptimising conformer: 100%|███████████████████████| 2/2 [00:05<00:00,  2.69s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-plip/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
Optimising conformer:  67%|███████████████▎       | 6/9 [00:52<00:33, 11.28s/it]Optimising conformer:  78%|█████████████████▉     | 7/9 [00:58<00:19,  9.61s/it]Optimising conformer:  89%|████████████████████▍  | 8/9 [01:00<00:07,  7.39s/it]Optimising conformer: 100%|███████████████████████| 9/9 [01:09<00:00,  7.63s/it]Optimising conformer: 100%|███████████████████████| 9/9 [01:09<00:00,  7.68s/it]
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-01-29 04:22:10,087 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Bonds with undefined stereochemistry are:
 - Bond 1 (atoms 1-2 of element (C-C)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-01-29 04:22:12,059 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Bonds with undefined stereochemistry are:
 - Bond 1 (atoms 1-2 of element (C-C)

TIME Completed the molecule generation in 118.6s.
TIME Completed the molecule generation in 146.0s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 29 conformers. 
Generated 25 conformers. 
Removed 24 conformers. 
Removed 23 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/1 [00:00<?, ?it/s]failed to equip `nnpops` with error: No module named 'NNPOps'

Optimising conformer:   0%|                               | 0/6 [00:00<?, ?it/s][AOptimising conformer: 100%|███████████████████████| 1/1 [00:04<00:00,  4.80s/it]Optimising conformer: 100%|███████████████████████| 1/1 [00:04<00:00,  4.81s/it]
2024-01-29 04:23:45,168 - distributed.worker - WARNING - Compute Failed
Key:       evaluate-b9242ad3-e47d-44b1-b86c-1e5c8f839db2
Function:  evaluate
args:      (<rdkit.Chem.rdchem.Mol object at 0x2b85ccd5e0e0>, 6, 'O=C(C(c1c(O[H])c2c(c(c(c(c2nc1[H])[H])[H])[H])[H])([H])[H])N(C1(c2c(c(c(nc2[H])[H])[H])[H])C(C1([H])[H])([H])[H])[H]', '/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-plip/rec_final.pdb', '/nobackup/nmb1063/sars-fegrow/gnina')
kwargs:    {}
Exception: "Exception('No Conformers')"

2024-01-29 04:23:45,168 [WARNING] [worker.py:2352] distributed.worker: Compute Failed
Key:       evaluate-b9242ad3-e47d-44b1-b86c-1e5c8f839db2
Function:  evaluate
args:      (<rdkit.Chem.rdchem.Mol object at 0x2b85ccd5e0e0>, 6, 'O=C(C(c1c(O[H])c2c(c(c(c(c2nc1[H])[H])[H])[H])[H])([H])[H])N(C1(c2c(c(c(nc2[H])[H])[H])[H])C(C1([H])[H])([H])[H])[H]', '/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-plip/rec_final.pdb', '/nobackup/nmb1063/sars-fegrow/gnina')
kwargs:    {}
Exception: "Exception('No Conformers')"


Optimising conformer:  17%|███▊                   | 1/6 [00:04<00:20,  4.19s/it][A
Optimising conformer:  33%|███████▋               | 2/6 [00:07<00:13,  3.41s/it][A
Optimising conformer:  50%|███████████▌           | 3/6 [00:08<00:07,  2.49s/it][A
Optimising conformer:  67%|███████████████▎       | 4/6 [00:15<00:08,  4.24s/it][A
Optimising conformer:  83%|███████████████████▏   | 5/6 [00:18<00:03,  3.72s/it][A
Optimising conformer: 100%|███████████████████████| 6/6 [00:20<00:00,  3.31s/it][AOptimising conformer: 100%|███████████████████████| 6/6 [00:20<00:00,  3.45s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-plip/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-01-29 05:07:09,200 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 8)

/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/parmed/structure.py:1775: UnitStrippedWarning: The unit of the quantity is stripped when downcasting to ndarray.
  coords = np.array(value, dtype=np.float64, copy=False, subok=True)
2024-01-29 05:07:11,147 [WARNING] [rdkit_wrapper.py:3213] openff.toolkit.utils.rdkit_wrapper: Warning (not error because allow_undefined_stereo=True): RDMol has unspecified stereochemistry. Undefined chiral centers are:
 - Atom C (index 8)

TIME Completed the molecule generation in 128.8s.
TIME changed dir: 0.0s
TIME changed dir: 0.0s
Generated 37 conformers. 
Removed 28 conformers. 
Generated 49 conformers. 
Removed 47 conformers. 
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/9 [00:00<?, ?it/s]Optimising conformer:  11%|██▌                    | 1/9 [00:03<00:25,  3.13s/it]Optimising conformer:  22%|█████                  | 2/9 [00:09<00:33,  4.81s/it]Optimising conformer:  33%|███████▋               | 3/9 [00:13<00:27,  4.55s/it]Optimising conformer:  44%|██████████▏            | 4/9 [00:18<00:23,  4.62s/it]Optimising conformer:  56%|████████████▊          | 5/9 [00:20<00:15,  3.78s/it]Optimising conformer:  67%|███████████████▎       | 6/9 [00:22<00:09,  3.23s/it]Optimising conformer:  78%|█████████████████▉     | 7/9 [00:30<00:09,  4.88s/it]Optimising conformer:  89%|████████████████████▍  | 8/9 [00:33<00:04,  4.06s/it]Optimising conformer: 100%|███████████████████████| 9/9 [00:39<00:00,  4.68s/it]Optimising conformer: 100%|███████████████████████| 9/9 [00:39<00:00,  4.35s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-plip/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
TIME Completed the molecule generation in 119.7s.
using ani2x
/mnt/storage/nobackup/nmb1063/mamba/envs/fegrow/lib/python3.9/site-packages/torchani/resources/
failed to equip `nnpops` with error: No module named 'NNPOps'
Optimising conformer:   0%|                               | 0/2 [00:00<?, ?it/s]Optimising conformer:  50%|███████████▌           | 1/2 [00:04<00:04,  4.23s/it]Optimising conformer: 100%|███████████████████████| 2/2 [00:08<00:00,  4.48s/it]Optimising conformer: 100%|███████████████████████| 2/2 [00:08<00:00,  4.44s/it]
/mnt/storage/nobackup/nmb1063/code/gal/sars-cov-2-main-protease-al-study-plip/rsearcher.py:89: UserWarning: The ligand was an array (SDF?). Using the first frame. 
  warnings.warn("The ligand was an array (SDF?). Using the first frame. ")
slurmstepd: error: *** JOB 19877697 ON sb048 CANCELLED AT 2024-01-29T05:30:07 ***
